internal/ci/ethereum_test

```diff
@@ -16,13 +16,25 @@ assert ()
   fi
 }
 
+waitForResponse ()
+{
+  sleepCount=0
+  while ! curl -s "$1" && [ "$sleepCount" -le "10" ]; do
```

Comment:
I would think you'd want to put the `sleepCount` first so it short circuits and doesn't do the `curl` one more time than you wanted.